### PR TITLE
Emote Page - Comments

### PR DESCRIPTION
**Pull request by sirinoks**
Smol styling of comments, can't connect to data cause no data yet.

### CODE REVIEW

**Comment by sirinoks** [comment]
Pls review and merge yourself, since it goes into your branch @AnatoleAM

File: src/views/EmotePage/EmoteComment.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export default defineComponent({
+	components: {
+		UserTag,
+	},
+});
+</script>
+
+<style lang="scss" scoped>
+@import "@scss/emotes.scss";
+@import "@scss/components/user-tag.scss";
```
Comment (AnatoleAM) [inline src/views/EmotePage/EmoteComment.vue:29]: Why was the the UserTag component's styling imported here?

**Review by AnatoleAM — CHANGES_REQUESTED** [review]

File: src/views/EmotePage/EmoteComment.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable prettier/prettier */
```
[the remark this answers is not in the document]
Comment (sirinoks) [reply src/views/EmotePage/EmoteComment.vue:1]: I think it's one of those auto fixes it did itself.

File: src/views/EmotePage/EmoteComment.vue (hunk 1 above)
Comment (sirinoks) [reply src/views/EmotePage/EmoteComment.vue:29]: user-tag:
Cause it uses UserTag in comments, but we can't really import the data for it to really check. For now it's hardcoded along with CSS, but when we have data, we will use the component.

emotes:
I'm actually not sure if you have to import it for the inner component separately. Does the child component use the parent's css? If that's the case, then yea, my bad.

Comment (sirinoks) [reply src/views/EmotePage/EmoteComment.vue:28]: Ok, tested, that is the case. I don't need those lines. Thanks!

**Review by AnatoleAM — APPROVED** [review]

**Pull request merged**